### ISSUE TITLE
No Emscripten frontend

### ISSUE THREAD

**Issue by AnyOldName3** [wareya/FunkshunL#1]
How am I supposed to use this on the web? I'll do *anything* to avoid writing JavaScript.

**Comment by wareya** [comment]
Good catch! Can't call it production-ready when you can't even cross-compile to asm.js.

**Comment by AnyOldName3** [comment]
WebAssembly these days. Emscripten actually compiles to regular JS if you tell it not to use WASM as vaguely-modern browsers transpile Asm.js to WASM before JITting it, and if you *don't* want WASM, you're probably targeting some seriously ancient browsers that predate Asm.js optimisations.
